test/unit_system_builder_test.rb

```diff
@@ -37,4 +37,76 @@ class Measured::UnitSystemBuilderTest < ActiveSupport::TestCase
 
     assert_equal 'BOLD', measurable.unit_system.unit_for!(:BOLD).name
   end
+
+  test "#si_unit adds 21 new units" do
+    measurable = Measured.build do
+      unit :ft
+      si_unit :in, value: "12 ft", aliases: [:ins]
+    end
+
+    assert_equal 22, measurable.unit_names.count
+  end
+
+  test "#si_unit creates units with si prefixes" do
+    measurable = Measured.build do
+      unit :in
+      si_unit :ft, value: "12 in", aliases: [:fts]
+    end
+
+    prefixes = Measured::UnitSystemBuilder::SI_PREFIXES.map {|short, _, _| "#{short}ft"}
+    prefixes += ['in', 'ft']
```

Comment:
Just a couple of small style nitpicks for the above two lines:

1. [Spaces around curly braces](https://github.com/bbatsov/ruby-style-guide#spaces-braces), and
2. [prefer `%w(...)` for arrays of strings](https://github.com/bbatsov/ruby-style-guide#percent-w).